examples/example-rtm-client-datastore.js

```diff
@@ -7,8 +7,8 @@ var MemoryDataStore = require('@slack/client').MemoryDataStore;
 var token = process.env.SLACK_API_TOKEN || '';
 
 var rtm = new RtmClient(token, {
-  logLevel: 'info', // check this out for more on logger: https://github.com/winstonjs/winston
-  dataStore: new MemoryDataStore({}) // pass a new MemoryDataStore instance to cache information
```

Comment:
This was done to make the example consistent with the code in the readme. Consistent examples give the user a better idea of what sensible defaults to use. The purpose of the PR is also to improve consistency.